pkg/publisher/util/utils.go

```diff
@@ -18,13 +18,7 @@ import (
 	"github.com/bacalhau-project/bacalhau/pkg/system"
 )
 
-/* TODO(forrest): Fix me(!):
-- method has wrong name, this make S3 publisher, and noop, in addition to IPFS
-Issue: https://github.com/bacalhau-project/bacalhau/issues/2555
-
-*/
-
-func NewIPFSPublishers(
+func NewPublisherProvider(
```

Comment:
To align with existing names of `*Providers` should this be `NewStandardPublisherProvider`?

Reply:
If there was going to be a non-standard version, it would be worth it, but think this is alright.